content/principles.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Example'
```

Comment:
Tittelen som vises i navbaren er "Example". Kanskje finne en annen tittel?

Reply:
Helt enig, en "copy and paste" glipp fra min side.

Reply:
da har jeg endret og pushet